JavaDirectory/src/main/java/com/bug/free/invention/api/controllers/US024.java

```diff
@@ -0,0 +1,33 @@
+package com.bug.free.invention.api.controllers;
+
+import org.json.JSONObject;
+import org.springframework.web.bind.annotation.*;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+@RestController
+@RequestMapping("/api/login")
+public class US024 {
+
+    @PostMapping(value = "/AuthLogin", consumes = "application/json")
+    public String validateLogin(@RequestBody String authDetails){
+        JSONObject userDetails = new JSONObject(authDetails);
+        String dbQuery = "SELECT `employeeID`,`isAdmin` FROM `employee` WHERE `username` = ? AND `userPassword` = ? LIMIT 1;";
+        try(Connection DatabaseConnection = DBConfig.getConnection()){
```

Comment:
Variables should not be Capitalised
```suggestion
        try(Connection databaseConnection = DBConfig.getConnection()){
```